components/mfa/mfa_controller/mfa_controller.jsx

```diff
@@ -95,9 +96,10 @@ export default class MFAController extends React.Component {
     }
 }
 
-MFAController.defaultProps = {
-};
 MFAController.propTypes = {
```

Comment:
We can move this to the class `static propTypes` attribute. It isn't a stopper for the PR to be merged, of course :)